packages/extension-ui/src/Popup/Authorize/index.tsx

```diff
@@ -6,18 +6,21 @@ import type { ThemeProps } from '../../types';
 import React, { useContext } from 'react';
 import styled from 'styled-components';
 
-import { AuthorizeReqContext, PopupBorderContainer } from '../../components';
-import useTranslation from '../../hooks/useTranslation';
-import { Header } from '../../partials';
+import { AccountContext, AuthorizeReqContext, PopupBorderContainer } from '../../components';
 import Request from './Request';
 
 interface Props extends ThemeProps {
   className?: string;
+  condition?: boolean;
 }
 
-function Authorize({ className = '' }: Props): React.ReactElement {
-  const { t } = useTranslation();
+function Authorize({ className = '', condition = false }: Props): React.ReactElement {
   const requests = useContext(AuthorizeReqContext);
+  const { accounts } = useContext(AccountContext);
+
+  // passing it as a prop to the styled component
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  condition = !accounts.length;
```

Comment:
I would prefer have `theme.newTransactionBackground` by default and a modifier class in styles that adds warning color and use `accounts.lenght` status to add it. Having a prop that you override here is an anti-pattern and looks like sort of a hack.